plugin_repackaging.sh

```
@@ -12,183 +12,155 @@ PIP_MIRROR_URL="${PIP_MIRROR_URL:-$DEFAULT_PIP_MIRROR_URL}"
+    [[ ! -f "$IGNORE_FILE" ]] && IGNORE_FILE=".gitignore"
+
+    if [[ -f "$IGNORE_FILE" ]]; then
+        if [[ "$OS_TYPE" == "linux" ]]; then
+            sed -i '/^wheels\//d' "$IGNORE_FILE"
+        else
+            sed -i ".bak" '/^wheels\//d' "$IGNORE_FILE"
+            rm -f "$IGNORE_FILE.bak"
+        fi
+    fi
+
+    cd "$CURR_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)